### PR TITLE
Update buy-ravencoin.md, to include foundational donations

### PR DESCRIPTION
**Pull request by ladams1776**
We want to let the users know that exchange fees are partly
donated back to the raven coin foundation

### CODE REVIEW

File: _pages/buy-ravencoin.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ permalink: /buy-ravencoin/
                   <div class="bg-grey-lighter max-w-sm rounded overflow-hidden shadow-md hover:by-grey">
                     <a class="block px-6 py-8" href="{{ exchange.url }}" target="_blank"><img src="{{ exchange.logo }}" alt="{{ exchange.name }} exchange"/></a>
                   </div>
+                  <span><i class="zmdi zmdi-balance"></i> &nbsp; Donates to foundation</span>
```
Comment (ladams1776) [inline _pages/buy-ravencoin.md:19]: perhaps the `&nbsp;` is overkill. I just wanted to ensure padding. not use to jekyll.

**Comment by ladams1776** [comment]
To address Issue: https://github.com/RavenProject/ravenproject.github.io/issues/67

File: _data/exchanges.yml [hunk 2]
```diff
@@ -2,173 +2,208 @@
...
   logo: /assets/img/pages/exchanges/vertbase.png
   url: https://app.vertbase.com/assets/rvn
   kyc: true
+  donates_to_project: true
 
 - name: StealthEX
   logo: /assets/img/pages/exchanges/stealthex.png
   url: https://stealthex.io/?to=rvn
   kyc: false
+  donates_to_project: false
```
Comment (ladams1776) [inline _data/exchanges.yml:209]: Do we want to add Dynx? I see the issue states Dynx as one of the exchanges that donates back to the foundation

**Comment by ladams1776** [comment]
Test Evidence:
Screen Shot 2021-03-29 at 6 03 15 PM

Screen Shot 2021-03-29 at 6 03 27 PM

File: _data/exchanges.yml (hunk 2 above)
Comment (jeroz1) [reply _data/exchanges.yml:215]: Dynx is gone

**Comment by jeroz1** [comment]
I like the changes a lot! 

Would it be possible to fix the Exchange title alignment? Currently, the badge pushes the name down. 

Come to think of it. Can we get rid of the exchange titles? This information is already in the logos themselves. Extra titles at the bottom of the logos seem pretty redundant to me. Let's do that in a separate PR.

**Comment by ladams1776** [comment]
> I like the changes a lot!
> 
> Would it be possible to fix the Exchange title alignment? Currently, the badge pushes the name down.
> 
> Come to think of it. Can we get rid of the exchange titles? This information is already in the logos themselves. Extra titles at the bottom of the logos seem pretty redundant to me. Let's do that in a separate PR.

I think this is what we are looking for.
Screen Shot 2021-04-03 at 10 19 02 PM

**Review by jeroz1 — CHANGES_REQUESTED** [review]
_data/exchanges.yml needs a new line at the end of the file

**Comment by ladams1776** [comment]
> _data/exchanges.yml needs a new line at the end of the file

ok, I think that should do it. Just updated it.

**Review by jeroz1 — APPROVED** [review]

**Pull request merged**